config/config-network.yaml

```diff
@@ -83,3 +83,13 @@ data:
     # update this value during the setup of Knative, to avoid getting
     # undefined behavior.
     clusteringress.class: "istio.ingress.networking.knative.dev"
+
+    # TLS mode of Knative. There are three TLS modes: "Manual", "Auto", and "Force".
+    # 1. "Manual" mode: No TLS certificates will be provisioned, and Gateway object will not be reconciled.
+    # Operator needs to manually provision TLS certificates and configure Gateway to support TLS.
+    # 2. "Auto" mode: TLS certificates will be provisioned, and Gateway object will be reconciled to terminate TLS connection.
+    # For a given host, both HTTP connection and HTTPS connection will be supported.
+    # 3. "Force" mode: TLS certificates will be provisioned, and Gateway object will be reconciled to terminate TLS connection.
+    # For a given host, HTTPS will be supported. And HTTP connection will be redirected to HTTPS connection. The major difference btw AUTO and FORCE is the HTTP redirection.
+    # If Istio is the ClusterIngress solution, then "Auto" and "Force" requires Istio 1.1 (or above)
```

Comment:
I would reword the above bearing in mind that these are user-facing docs, and users don't really know/care about reconciliation.  Also bear in mind that this is `config-network`, not `config-istio`, so we should probably remain agnostic of the networking layer here.

Reply:
Rephrased this file to avoid mentioning any istio specific details.